tests/test_store.py

```diff
@@ -214,6 +220,34 @@ def test_view_query(
         # tests that the query was valid and successful, that a view subset has actually been retrived
         assert synapse_store_special_scope.storageFileviewTable.empty is False
 
+    @pytest.mark.parametrize(
+        "asset_view,columns,expectation",
+        [
+            ("syn62339865", ["path"], pytest.raises(ValueError)),
+            ("syn62340177", ["id"], pytest.raises(ValueError)),
+            ("syn23643253", ["id", "path"], does_not_raise()),
+        ],
+    )
+    def test_view_query_exception(
+        self,
+        synapse_store_special_scope: SynapseStorage,
+        asset_view: str,
+        columns: list,
+        expectation: str,
+    ) -> None:
+        project_scope = ["syn23643250"]
+
+        # set to use appropriate test view and set scope
+        CONFIG.synapse_master_fileview_id = asset_view
+        synapse_store_special_scope.project_scope = project_scope
+
+        # ensure approriate exception is raised
+        with expectation:
```

Comment:
it is a small thing, but I imagine you could do: 
```
    try: 
          with expectation: 
                  XXX
    finally:
        CONFIG.synapse_master_fileview_id = "syn23643253"
```
More about using `finally`: https://www.w3schools.com/python/ref_keyword_finally.asp#:~:text=It%20defines%20a%20block%20of,objects%20and%20clean%20up%20resources.